src/app/shared/object-list/object-list.component.ts

```diff
@@ -54,6 +58,11 @@ export class ObjectListComponent {
    * Whether or not the pager is visible when there is only a single page of results
    */
   @Input() hidePagerWhenSinglePage = true;
+  @Input() selectable = false;
+  @Input() selectionConfig: { repeatable: boolean, listId: string };
+  // @Input() previousSelection: ListableObject[] = [];
+  // allSelected = false;
+  // selectAllLoading = false;
```

Comment:
I'm not sure why these 3 commented lines are here? Are they a mistake?

Reply:
This is just a remnant of how thing previously worked. I removed it.